doc/source/howto/immigrate.rst

```diff
@@ -0,0 +1,66 @@
+How to import non-AiiDA VASP runs
+=================================
+
+Basic process
+-------------
+
+The following will assume that you have
+
+ * AiiDA and AiiDA-VASP installed and a profile set up
+ * added ``<computer>`` to your profile
+ * added a VASP code (does not have to be runnable in principle) with computer ``<computer>`` to your profile
+ * a working understanding of how to run calculations in AiiDA
+
+See :ref:`main-quickstart` for an overview of setting up AiiDA-VASP and :ref:`main-running` for a quick intro to running calculations.
+
+Assuming you ran VASP in ``/scratch/<user>/<runfolder>/`` on the remote (or local) computer ``<computer>``, the following python snippets will help you integrate that run into your AiiDA profile database::
+
+   from aiida.orm import CalculationFactory, Code
+   from aiida.work import run
+
+   calculation_folder = '/scratch/<user>/<runfolder>/'
+   # first, get the code which the calculation will use
+   code = Code.get_from_string('vasp@<computer>')  # use the name of your code
+
+   # then comes the information which can not be read from input files
+   resources = {'num_machines': 1, 'num_mpiprocs_per_machine': 16}  # whatever is appropriate for you
```

Comment:
Why do we need this? Provenance? Can we not just set this to None if we have no idea. In fact this happens quite often.

Reply:
You can try setting it to None, but I believe AiiDA will refuse to run it.

Reply:
Of course, this is only to get it through the `run`/`submit`. That is fine.